project/plugins.sbt

```diff
@@ -8,11 +8,11 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 
```

Comment:
Also,  please bump `sbt-accessibility-linter` to [0.36.0](https://github.com/hmrc/sbt-accessibility-linter/releases/tag/v0.36.0), thanks.